chat.js

```diff
@@ -745,6 +745,14 @@ class CommandContext {
 				return false;
 			}
 
+			// If the corresponding config option is set, non-AC users cannot send links, except to staff.
+			if (Config.restrictLinks && !user.autoconfirmed && message.match(linkRegex)) {
+				if (!(targetUser && targetUser.can('lock'))) {
+					this.errorReply("Your account must be autoconfirmed to send links to other users, except for staff.");
```

Comment:
I think this should say "except for global staff"

Reply:
good point